e2e-tests/functions

```diff
@@ -312,6 +345,9 @@ deploy_chaos_mesh() {
 
 	helm repo add chaos-mesh https://charts.chaos-mesh.org
 	helm install chaos-mesh chaos-mesh/chaos-mesh --namespace=${NAMESPACE} --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock --set dashboard.create=false --version 2.5.1
+	if [ ! -z "$OPENSHIFT" ]; then
```

Comment:
We can use `if [[ $OPENSHIFT ]];` like above or also `if [ -n "$OPENSHIFT" ]` instead of negation.